libnd4j/include/execution/impl/ThreadPool.cpp

```diff
@@ -78,7 +78,6 @@ ThreadPool::ThreadPool() {
   _threads.resize(_available.load());
   _interfaces.resize(_available.load());
 
-#ifndef __NEC__
```

Comment:
The corresponding endif is left over and the comment that follows this ifndef also neds to be removed